.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,63 @@
+<!--
+BEFORE YOU CREATE A PULL REQUEST:
+
+Ensure you have read over [CONTRIBUTING.md](./CONTRIBUTING.md). We provide VERY defined guidance (as such, we strongly adhere to it).
+
+A summary of our expectations:
+ - You are not submitting a pull request from your MASTER / MAIN branch.
+ - You are able to sign the Contributor License Agreement (CLA).
+ - YOUR GIT COMMIT MESSAGE FORMAT IS EXTREMELY IMPORTANT. We have a very defined expectation for this format and are sticklers about it. Really, READ the entire Contributing document. It will save you and us pain.
+ - Do not reformat code, it makes it hard to see what has changed. Leave the formatting to us.
+
+THANKS! We appreciate you reading the entire Contributing document and not just scanning through it.
+
+Name your issue appropriately: give it a sentence that reads well enough for anyone seeing this in release notes to know what the issue is.

```

Comment:
Shouldn't this line be in the issue template, instead of the pull request template?

Reply:
Agreed. Fixed.

Reply:
And added it to the issue template.